packages/webapp/src/components/Animals/FixedHeaderContainer/styles.module.scss

```diff
@@ -37,3 +37,14 @@
     border: 1px solid var(--Colors-Primary-Primary-teal-50);
   }
 }
+
+.overflowStyle {
+  display: flex;
+  justify-content: center;
+  overflow-y: scroll;
+}
+
+.childrenWrapper {
+  width: 100%;
+  max-width: 1024px;
```

Comment:
Can this be set by the children instead or passed in a prop? The [AddAnimalsSummary](https://www.figma.com/file/eMffC0h3JPlWtzNttxJpil/Litefarm-Updated?node-id=12490%3A123487&mode=dev) is true full-width on desktop, and it looks like even the [details forms](https://www.figma.com/file/eMffC0h3JPlWtzNttxJpil/Litefarm-Updated?node-id=5023%3A52395&mode=dev) are designed at greater than 1024px width.

Reply:
The max-width of the content of the AddAnimalsSummary is `1024px`, correct? I will update `DivWrapper` to take `classes` so that we can add `background-color`(white) to the wrapper div. It will look like this:

https://github.com/LiteFarmOrg/LiteFarm/assets/33141219/ad9b2b0a-a05d-49b6-a492-30e8db134692

I think details forms' max-width is 1024px, I asked Loïc about animal basics view whose mock up had a width greater than 1024px, but max-width was 1024px...
Let me confirm!